components/src/core/drawer/drawer-body/nav-item/drawer-nav-item.component.scss

```diff
@@ -33,6 +36,13 @@ pxb-drawer-nav-item {
     position: relative;
     user-select: none;
 
+    &.pxb-drawer-nav-item-compact .pxb-info-list-item-content {
+        &.pxb-info-list-item-dense,
+        .mat-list-item-content {
+            height: 40px !important;
```

Comment:
@huayunh correct me if I'm wrong, but I believe the compact version is also supposed to have a margin between drawer items.

Reply:
the compact height is 40px total.

Also I've added in a conditional that removes the subtitle from a navItem if the compact density is used.